docs/user/custom-resources/ratelimit/04-00-ratelimit.md

```diff
@@ -0,0 +1,64 @@
+| **selectorLabels**                    | **YES**  | Labels that specify the set of Pods to which the configuration applies.<br/> Each Pod can match only one RateLimit CR.<br/> The label scope is limited to the namespace where the resource is located.                                                                                                                                                                                             |
+| **local**                             | **YES**  | Local rate limit configuration.                                                                                                                                                                                                                                                                                                                                                                    |
+| **local.defaultBucket**               | **YES**  | The default token bucket for rate limiting requests. <br/>If additional local buckets are configured in the same RateLimit CR, this bucket serves as a fallback for requests that don't match any other bucket's criteria. <br/>Each request consumes a single token. If a token is available, the request is allowed. If no tokens are available, the request is rejected with status code `429`. |
+| **local.defaultBucket.maxTokens**     | **YES**  | The maximum tokens that the bucket can hold. This is also the number of tokens that the bucket initially contains.                                                                                                                                                                                                                                                                                 |
+| **local.defaultBucket.tokensPerFill** | **YES**  | The number of tokens added to the bucket during each fill interval.                                                                                                                                                                                                                                                                                                                                |
+| **local.defaultBucket.fillInterval**  | **YES**  | The fill interval that tokens are added to the bucket. <br/>During each fill interval, `tokensPerFill` are added to the bucket. The bucket will never contain more than `maxTokens` tokens. The `fillInterval` must be greater than or equal to 50ms to avoid excessive refills.                                                                                                                   |
```

Comment:
```suggestion
| **local.defaultBucket.fillInterval**  | **YES**  | The fill interval during which tokens are added to the bucket. <br/>During each fill interval, `tokensPerFill` are added to the bucket. The bucket will never contain more than `maxTokens` tokens. The `fillInterval` must be greater than or equal to 50ms to avoid excessive refills.                                                                                                                   |
```